.github/workflows/SQL2022.yml

```diff
@@ -0,0 +1,158 @@
+      
+      - name: Alter AirlineUser
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -l 5 -Q "
+          USE AirlineTestDB;
+          ALTER USER [AirlineUser] WITH LOGIN=[AirlineUser]"
+      
+      - name: Alter AirlineUserdbowner
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -l 5 -Q "
+          USE AirlineTestDB;
+          ALTER USER [AirlineUserdbowner] WITH LOGIN=[AirlineUserdbowner]"
+
+      - name: Alter Authorization
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -l 5 -Q "
+          USE AirlineTestDB;
+          ALTER AUTHORIZATION ON SCHEMA::[db_owner] TO [AirlineUserdbowner]"
+
+      - name: Install R for MLS
+        run: |
+          curl -L -o R-4.2.0-win.exe https://cloud.r-project.org/bin/windows/base/old/4.2.0/R-4.2.0-win.exe
+          R-4.2.0-win.exe /VERYSILENT /DIR=C:\MLS\R
+
+      - name: Install CompatibilityAPI and RevoScaleR after installing their dependencies (iterators, foreach, R6, and jsonlite).
+        run: |
+          C:\MLS\R\bin\Rscript.exe -e "install.packages(c('iterators', 'foreach', 'R6', 'jsonlite'), repos='https://cloud.r-project.org/', lib='C:\\MLS\\R\\library')"
+          C:\MLS\R\bin\Rscript.exe -e "install.packages('https://aka.ms/sqlml/r4.2/windows/CompatibilityAPI_1.1.0.zip', repos=NULL, lib='C:\\MLS\\R\\library')"
+          C:\MLS\R\bin\Rscript.exe -e "install.packages('https://aka.ms/sqlml/r4.2/windows/RevoScaleR_10.0.1.zip', repos=NULL, lib='C:\\MLS\\R\\library')"
+          C:\MLS\R\bin\Rscript.exe -e "install.packages(c('rtools'), repo='https://cloud.r-project.org/', lib='C:\\MLS\\R\\library')"
+
+
+      - name: Configure the R runtime installed for MLS with SQL Server
+        run: C:\MLS\R\library\RevoScaleR\rxLibs\x64\RegisterRext.exe /configure /rhome:"C:\MLS\R" /instance:"MSSQLSERVER"
+
+      - name: Enable External Scripts
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -Q "EXEC sp_configure  'external scripts enabled', 1;"
+
+      - name: Reconfigure
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -Q "RECONFIGURE WITH OVERRIDE;"  
+
+      - name: Restart the Service
+        run: |
+          sqlcmd -S localhost -U SA -P %dbPassword% -Q "SHUTDOWN WITH NOWAIT"
+          timeout /t 5 /nobreak
+          net start "MSSQLSERVER"
+
+      - name: Execute SPEES for R
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -l 5 -Q "
+          EXEC sp_execute_external_script  @language =N'R',
+          @script=N'
+          OutputDataSet <- InputDataSet;
+          ',
+          @input_data_1 =N'SELECT 1 AS hello'
+          WITH RESULT SETS (([hello] int not null));"
+
+      - name: Set up R ${{ env.r-version }} Runtime
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: ${{ env.r-version }}
+
+      - name: Install R Package Dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          cache-version: 2
+          working-directory: ./R
+          extra-packages:
```

Comment:
does this github action allow you to explicitly define which R library these dependencies are installed in? 

This seems to be working fine since some tests pass. just curious.

Reply:
I did not see a way to do this in the documentation

https://github.com/r-lib/actions/tree/v2/setup-r-dependencies